test/utils/pickers-utils.tsx

```diff
@@ -1,34 +1,48 @@
 import * as React from 'react';
-import { parseISO } from 'date-fns';
 import { createRenderer, screen, RenderOptions, userEvent } from '@mui/monorepo/test/utils';
 import { CreateRendererOptions } from '@mui/monorepo/test/utils/createRenderer';
 import { TransitionProps } from '@mui/material/transitions';
 import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { AdapterLuxon } from '@mui/x-date-pickers/AdapterLuxon';
+import { AdapterMoment } from '@mui/x-date-pickers/AdapterMoment';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 
-// TODO make possible to pass here any utils using cli
-/**
- * Wrapper around `@date-io/date-fns` that resolves https://github.com/dmtrKovalenko/date-io/issues/479.
- * We're not using `adapter.date` in the implementation which means the implementation is safe.
- * But we do use it in tests where usage of ISO dates without timezone is problematic
- */
-export class AdapterClassToUse extends AdapterDateFns {
-  // Inlined AdapterDateFns#date which is not an instance method but instance property
-  // eslint-disable-next-line class-methods-use-this
-  date = (value?: any): Date => {
-    if (typeof value === 'string') {
-      return parseISO(value);
-    }
-    if (typeof value === 'undefined') {
-      return new Date();
-    }
-    if (value === null) {
-      // @ts-expect-error AdapterDateFns#date says it returns NotNullable but that's not true
-      return null;
-    }
-    return new Date(value);
-  };
+const availableAdapters = ['date-fns', 'day-js', 'luxon', 'moment'];
+let adapter = 'date-fns';
+
+// Check if we are in unit tests
+if (typeof window === 'undefined') {
```

Comment:
```suggestion
if (/jsdom/.test(window.navigator.userAgent)) {
```

Should we check for jsdom instead?
When I run `yarn test:unit` locally, `typeof window === 'undefined'` returns false

Reply:
That would be nice, but for another PR Could be nice to make test pass with other libraries on master 
And before that we should make unit test pass with the current code base which not the case

Reply:
Yeah, it can be addressed in separate PR

> And before that we should make unit test pass with the current code base which not the case

You mean to make unit tests pass with each of the adapters?

Reply:
Yes. For example some tests do not pass because some libraries use uppercase `AM/PM` and others use `am/pm`